WalletWasabi.Gui/Global.cs

```diff
@@ -462,7 +462,14 @@ public static async Task DisposeInWalletDependentServicesAsync()
 			{
 				WalletService.Coins.CollectionChanged -= Coins_CollectionChanged;
 			}
-			CancelWalletServiceInitialization?.Cancel();
+			try
+			{
+				CancelWalletServiceInitialization.Cancel();
+			}
+			catch(ObjectDisposedException)
+			{
+				Logger.LogWarning($"{nameof(CancelWalletServiceInitialization)} is disposed. This can occur due to an error while processing the wallet.");
```

Comment:
```suggestion
				Logger.LogWarning($"{nameof(CancelWalletServiceInitialization)} is disposed. This can occur due to an error while processing the wallet.", nameof(Global));
```